src/core/structures/PDFAcroForm.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  PDFArray,
+  PDFBool,
+  PDFContext,
+  PDFDict,
+  PDFName,
+  PDFNumber,
+  PDFObject,
+  PDFStream,
+  PDFString,
+} from 'src/core';
+
+class PDFAcroForm extends PDFDict {
+  static fromMapWithContext(
+    dict: Map<PDFName, PDFObject>,

```

Comment:
I think it would be best to use the `DictMap` type here, instead of redefining `Map<PDFName, PDFObject>`:

https://github.com/Hopding/pdf-lib/blob/620747a044c2b3864c9d844b7c67b924b4f94ea5/src/core/objects/PDFDict.ts#L14

Reply:
Updated